Main_project_Helpers.rb

```diff
@@ -100,18 +91,49 @@ def create_new_issue(issue_data)
  		issue
 	end	
 
+
 	def open_project(project)
-		@driver.find_element(:css, '.projects').click
-		@driver.find_element(:css, '#q').send_keys project.name
-		@driver.action.send_keys(:enter).perform
-		available_projects = @driver.find_elements(:css, '.project .highlight'). map {|element| element.text}
+		@driver.navigate.to project.get_url
+	end
+
+
+	def open_issues_tab(project)
+		open_project(project)
+		@wait.until {@driver.find_element(:css, '.issues').displayed?}
+		@driver.find_element(:css, '.issues').click
+	end
 
 
+	def is_issues_tab_empty?(project)
+		open_issues_tab(project)
+		if @driver.find_element(:css, '.nodata').displayed?
+			return true
+		end
+
+	rescue Selenium::WebDriver::Error::NoSuchElementError => e
+		puts 'Exception ' + e.class.to_s + ' issues tab was not empty'
+		return false
+    end
+
+
+	def create_or_not_create_new_issue(project)
+		open_project(project)
+		random_action = rand(0..1)
+		if random_action == 1
+			issue = create_new_issue({})
+			open_issues_tab(project)
```

Comment:
if you have this line in both ```if``` and ```else``` blocks please place it after ```if``` statement
you won't need ```else``` block at all then